go.mod

```diff
@@ -2,4 +2,4 @@ module github.com/prebid/go-gpp
 
 go 1.16
 
-require github.com/stretchr/testify v1.8.1 // indirect
+require github.com/stretchr/testify v1.8.1
```

Comment:
This is actually used directly by tests. Ran `go mod tidy` to fix the annotation.